services/user_interface/main.py

```diff
@@ -1,41 +1,57 @@
 # coding: utf-8
 
-from flask import Flask
-from flask import render_template, redirect, url_for, request
 import json
+import os
+import requests
 
+from flask import Flask
+from flask import render_template, request
 
 app = Flask(__name__)
 app.debug = True
 app.secret_key = 'development'
 
+PERSISTENCE_HTTP_PORT = os.environ.get('PERSISTENCE_HTTP_PORT')
+HTTP_PORT = int(os.environ.get('HTTP_PORT'))
+
 
 @app.route('/user_labels', methods=['POST'])
```

Comment:
ja, da war ich mir auch nich sicher, welches modell man trainieren soll: manifesto labels oder left/right/neutral. 

man kann auf den manifesto labels trainieren und dann aber intern nur left/right/neutral beruecksichtigen. das haette den vorteil, dass man mit demselben modell left/right aber auch politische topics (wirtschaft, inneres, etc) vorhersagen kann. letzteres koennte hilfreich sein um themen zu sortieren. 

fuers erste is aber ok, einfach ganz am anfang manifesto labels auf left/right/neutral zu mappen, beim laden der daten schon, oder?

Reply:
Wäre es dann okay wenn ich beim Import das einfach auf links rechts Mitte mappe?

Wir müssen uns nochmal überlegen wie wir die Daten am besten speichern glaube ich, auch weil wir ja eventuell verschiedene Modelle gleichzeitig haben wollen oder?

Reply:
Ja, beim Import auf links/rechts/neutral Mappen wär super, so wie in dem Experimente File auch.

Reply:
Ok werde das beim Import hinzufügen, dann sind User und Manifesto labels in der Tabelle schonmal einheitlich.